components/kyma-environment-broker/internal/provider/aws_provider.go

```diff
@@ -79,20 +88,114 @@ func ZoneForAWSRegion(region string) string {
 	if !found {
 		zones = "a"
 	}
+	rand.Seed(time.Now().UnixNano())
```

Comment:
nit: I think `rand.Seed` is usually called only once, mostly in `init()`. Although I'm not sure if it's for anything else but performance reasons and if it has any effect on entropy at all if called once vs. each time before getting a random number.